### ISSUE TITLE
Harmonic not shown in Braille Output

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#34237]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Most other commonly used articulations, ornaments, bow markings, etc are now being shown in Braille.  The harmonic is not.

STEPS:
1.  Add a note to a score and then from the Articulation palette select harmonic.
2. Tab to the Braille panel and note that there is no sign added to the note to show it is a harmonic.  This should be Brailled as dots 13 after the note and then the associated finger number (usually finger 3 or 4 for example in violin music).

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.4

### Regression

No.

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
